server/pbench/bin/index-pbench

```diff
@@ -287,15 +291,19 @@ class ResultData(object):
                 self.counters['not_valid_json_file'] += 1
                 continue
 
-            convert_to_float(results)
+            # for each file, generate a sequence of documents and yield each one
+            for md, ts, src in gen_sources(results):
```

Comment:
Do we need the class name here to reference `gen_sources` as a static method?

Reply:
I don't know if we need to (I have to do some testing of index-pbench with this incorporated and see if it fails), but it's probably a good idea for code readability.

Reply:
And, yes, we *do* need to qualify the name.